frontend/packages/dev-console/integration-tests/features/monitoring/monitoring-Tab-Sidebar-ODC-4116.feature

```diff
@@ -0,0 +1,133 @@
+    Given user is at Topology page
+    And user has a workload of Deployment type resource
+    When user clicks on the workload to open the sidebar
+    And user clicks on Monitoring tab
+    Then user will see View Monitoring dashborad link
+    And user will see CPU Usage Metrics
+    And user will see Memory Usage Metrics
+    And user will see Receive Bandwidth Metrics
+    And user will see All Events dropdown
+
+
+@smoke
+Scenario: Monitoring tab on the Sidebar for Deployments
+    Given user is at Topology page
+    And user has a workload of Deployment type resource
+    When user clicks on the workload to open the sidebar
+    And user clicks on Monitoring tab
+    And user clicks on View Monitoring dashborad link
+    Then user will be taken to Dashboard tab on the Monitoring page
+
+
+@smoke
+Scenario: Health Checks option in Advanced Options: MH-04-TC01
+    Given user is at +Add page
+    When user clicks on From Git card
+    Then user will see Health Checks advanced option
+
+
+@smoke
+Scenario: Health Checks option in Advanced Options: MH-04-TC02
+    Given user is at +Add page
+    When user clicks on From Git card
+    And user enters Git URL
+    And user selects Builder Image
+    And user enters name of the application
+    And user selects Deployment as a resource type
+    And user clicks on Health Checks advanced option
+    And user clicks on Add Readiness Probe
+    And user clicks on right button
+    And user clicks on Add Liveness Probe
+    And user clicks on right button
+    And user clicks on Add Startup Probe
+    And user clicks on right button
+    And user clicks on Create button
+    Then user will be redirected to Topology page
+    And user will not see Add Health Checks link on the Sidebar
+    And user will see Readiness Probe added on the Add Health Checks page
+
+
+@smoke
+Scenario: Add Health Checks page: MH-05-TC01
+    Given user is at Topology page
+    And user has a new workload of Deployment type resource
+    When user clicks on the workload to open the sidebar
+    And user clicks on Add Health Checks
+    Then user will be redirected Add Health Checks page
+
+
+@smoke
+Scenario: Add Readiness Probe Health Check to Deployments from Sidebar: MH-05-TC02
+    Given user is at Topology page
+    And user has a new workload of Deployment type resource with no Health Checks added
+    When user clicks on the workload to open the sidebar
+    And user clicks on Add Health Checks
+    And user clicks on Add Readiness Probe
+    And user clicks on right button
+    And user clicks on Add button
+    Then user will be redirected to Topology page
+    And user will see Readiness Probe added on the Add Health Checks page
+
+
+@smoke
+Scenario: Add Liveness Probe Health Check to Deployments from Actions dropdown Sidebar: MH-05-TC02
+    Given user is at Topology page
+    And user has a new workload of Deployment Config type resource with no Health Checks added
```

Comment:
Scenario mentions Deployments and here it has Deployment-Config

Reply:
Updated